src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -6,6 +6,8 @@
 from azure.cli.core.util import CLIError
 from azure.cli.core.commands.parameters import get_resources_in_subscription
 
+from azure.mgmt.containerregistry.v2017_03_01.models import SkuTier
```

Comment:
We are using multiple api-version SDK to direct registry creation calls to different versions based on SKU. Basically, `Basic` SKU will go to `2017-03-01` version and `Managed` SKUs will go to `2017-06-01-preview`. At this point there is no need to expose multiple api-versions to users and the user specified SKU is enough to determine which api-version to go.